test/iris/expected_iris_hk.py

```diff
@@ -0,0 +1,34 @@
+'''
+ * Copyright (C) 2022  University of Alberta
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; either version 2
+ * of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+'''
+'''
+ * @file expected_iris_hk.py
+ * @author Jenish Patel
+ * @date 2022-08-08
+'''
+
+expected_iris_HK = {
```

Comment:
Are these simulated results or do we expect a range or something from real hardware? Is there a way to switch between them? Or is that unit test versus system test?